.github/workflows/examples-java-push.yml

```diff
@@ -55,3 +55,24 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+  deploy:
+    environment: sandbox
+    runs-on: ubuntu-18.04
+    needs: test
+#    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We could upload the manifest file in the test job and download it here, rather than check out the whole repo twice

Reply:
I think that's a good shout - it also makes sure that we're treating it all as one immutable thing